optuna_dashboard/ts/components/TrialList.tsx

```diff
@@ -437,6 +445,130 @@ const TrialArtifact: FC<{ trial: Trial }> = ({ trial }) => {
                 </CardContent>
               </Card>
             )
+          } else if (
+            modelViewerTargetExt.includes(a.filename.split(".").pop() || "")
```

Comment:
How about writing this condition like below to make the code more intuitive?

```suggestion
            a.filename.endswith(".stl") || a.filename.endswith(".3dm")
```

Reply:
When increasing the number of supported formats in the future, I thought it would be easier to understand if they were all in one place, so I made a check (modelViewerTargetExt.includes) to see if they were included in the array.
However, since there are only a few more formats to be added, I'm going to modify the comment you made.